go.mod

```diff
@@ -1,12 +1,14 @@
 module github.com/moby/buildkit
 
-go 1.22.0
+go 1.23.0
```

Comment:
> Hum this bump looks related to update of stargz-snapshotter to v0.16.0 for containerd v2 support

Yes, I cannot keep this at 1.22.0 because

````
$ go mod tidy --go=1.22.0
go: github.com/containerd/stargz-snapshotter@v0.16.0 requires go@1.23.0, but 1.22.0 is requested
```

Reply:
Changed to 1.22.7 in our `go.mod`, waiting for decision whether we're keeping it this way or stargz-snapshotter will do a downgrade to 1.22.0.

Reply:
@crazy-max @slonopotamus Thanks for notifying me. https://github.com/containerd/stargz-snapshotter/pull/1877 will downgrade `go` to `v1.22.0`. I'll release v0.16.2 after that PR is merged.

Reply:
Merged and released that patch as v0.16.2 . https://github.com/containerd/stargz-snapshotter/releases/tag/v0.16.2  Please try this release.

Reply:
Thanks, pulled in v0.16.2. Now our `go.mod` has 1.22.0, so this PR does not change required Go version.

Reply:
Thanks both!